cpp_format_tools/format.sh

```diff
@@ -18,8 +18,8 @@ fi
 # No CRLF line endings, except Windows files.
 "${SED[@]}" 's/\r$//' $($FIND -name '*.ps1' -prune -o \
   -name '*.cmd' -prune -o -type f -print)
-# No trailing spaces.
-"${SED[@]}" 's/ \+$//' $($FIND -type f -print)
+# No trailing spaces, except in patch.
+"${SED[@]}" 's/ \+$//' $($FIND -name "*.patch" -prune -o -type f -print)
```

Comment:
Apply the same for "*.diff" file?

Reply:
We do not have currently *.diff files in the repo, but sure, good idea.

Will add the same for diff.